src/utils/KmlUtils.js

```diff
@@ -0,0 +1,41 @@
+

```

Comment:
It looks like this file, uses either 4-spaces or tabs to indent, can you convert to 2 spaces?